csrankings-p.csv

```diff
@@ -94,6 +94,7 @@ Pangfeng Liu,National Taiwan University,https://www.coursera.org/instructor/~792
 Pankaj Jalote,IIIT Delhi,https://www.iiitd.edu.in/~jalote,NOSCHOLARPAGE
 Pankaj K. Agarwal,Duke University,https://users.cs.duke.edu/~pankaj,xe0eVksAAAAJ
 Pankaj Kumar Agarwal,Duke University,https://users.cs.duke.edu/~pankaj,xe0eVksAAAAJ
+Pankaj Mehra,Univ. of California - Santa Cruz,http://www.facebook.com/StrategyByMehra,z1jdSmoAAAAJ

```

Comment:
No evidence of UCSC affiliation on web page or https://www.linkedin.com/in/pankajmehra/.